pkg/pxc/service.go

```diff
@@ -85,3 +86,28 @@ func NewServiceProxySQL(cr *api.PerconaXtraDBCluster) *corev1.Service {
 
 	return obj
 }
+
+func NewPodDistributedBudget(cr *api.PerconaXtraDBCluster, pdbSpec *policyv1beta1.PodDisruptionBudgetSpec, componentName string) *policyv1beta1.PodDisruptionBudget {
+	component := cr.Name + "-" + appName + componentName
+
+	pdbSpec.Selector = &metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			"app":       appName,
+			"cluster":   cr.Name,
+			"component": component,
+		},
```

Comment:
`v1alpha1.App` interface has method `Lables()`. Use it instead of setting `MatchLabels` manually

Reply:
CR.Labels() were empty so I did it manually. But I agree it looks ugly.
Thanks.